dynawo/sources/Models/Modelica/Dynawo/Electrical/Loads/LoadAlphaBetaMotor.mo

```diff
@@ -0,0 +1,108 @@
+within Dynawo.Electrical.Loads;
+
+model LoadAlphaBetaMotor "AlphaBeta load in parallel with an induction motor. The load torque is supposed constant (does not change with rotor speed)."
+/*
+                  isPu          umPu  irPu
+    terminal --.--->----Rs+jXs---.----->--Rr+jXr------.
+               |                 |                    |
+            PQ load             jXm                Rr(1-s)/s
+               |                 |                    |
+               ----------------------------------------
+
+imPu goes downwards through jXm
+*/
+  import Modelica;
+  import Dynawo.Connectors;
+  import Dynawo.Electrical.SystemBase;
+  import Dynawo.Electrical.Controls.Basics.SwitchOff;
+
+  extends BaseClasses.BaseLoad;
+  extends AdditionalIcons.Load;
+
+  Connectors.ImPin omegaPu(value(start = SystemBase.omega0Pu)) "Network angular reference frequency in p.u (base OmegaNom)";
+
+  parameter Types.ApparentPowerModule PNom "Nominal active power consumed by a single motor (Unom base)";
+  parameter Types.PerUnit ActiveMotorShare "Share of active power consumed by motors in pu";
+  parameter Types.PerUnit RsPu "Stator resistance in pu (PNom base)";
+  parameter Types.PerUnit RrPu "Rotor resistance in pu (PNom base)";
+  parameter Types.PerUnit XsPu "Stator leakage reactance in pu (PNom base)";
+  parameter Types.PerUnit XrPu "Rotor leakage reactance in pu (PNom base)";
+  parameter Types.PerUnit XmPu "Magnetizing reactance in pu (PNom base)";
+  parameter Real H "Inertia constant (s, PNom base)";
+
+  parameter Real alpha "Active load sensitivity to voltage";
+  parameter Real beta "Reactive load sensitivity to voltage";
+
+  Types.ComplexCurrentPu isPu(re(start = is0Pu.re), im(start = is0Pu.im)) "Stator current in pu (Unom, PNom base)";
```

Comment:
base UNom, PNom (or SNom ?)

Reply:
Also, for a current, please give the convention (generator convention) or (receptor convention), thanks !
Same for imPu and irPu